.github/workflows/lint.yml

```diff
@@ -11,51 +11,31 @@ on:
       - main
       - feature-serverless-framework-deployment
 
-
-permissions:
-  contents: read
-  # Optional: allow read access to pull request. Use with `only-new-issues` option.
-  # pull-requests: read
-
 jobs:
   golangci:
-    name: lint
-    runs-on: ubuntu-22.04
+    name: GolangCI Lint
+    runs-on: self-hosted
+    env:
+      working-directory: ./src
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v4
-        with:
-          go-version: '1.20'
-          cache: false
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+      - name: Setup go
+        uses: actions/setup-go@v3
         with:
-          # Require: The version of golangci-lint to use.
-          # When `install-mode` is `binary` (default) the value can be v1.2 or v1.2.3 or `latest` to use the latest version.
-          # When `install-mode` is `goinstall` the value can be v1.2.3, `latest`, or the hash of a commit.
-          version: v1.53
-
-          # Optional: working directory, useful for monorepos
-          # working-directory: somedir
-
-          # Optional: golangci-lint command line arguments.
-          #
-          # Note: By default, the `.golangci.yml` file should be at the root of the repository.
-          # The location of the configuration file can be changed by using `--config=`
-          # args: --timeout=30m --config=/my/path/.golangci.yml --issues-exit-code=0
+          go-version: 1.19
 
-          # Optional: show only new issues if it's a pull request. The default value is `false`.
-          # only-new-issues: true
+      - name: Install dependencies
+        run: sudo apt update && sudo apt install -y libsnmp-dev build-essential zip docker.io
 
-          # Optional: if set to true, then all caching functionality will be completely disabled,
-          #           takes precedence over all other caching options.
-          # skip-cache: true
-
-          # Optional: if set to true, then the action won't cache or restore ~/go/pkg.
-          # skip-pkg-cache: true
-
-          # Optional: if set to true, then the action won't cache or restore ~/.cache/go-build.
-          # skip-build-cache: true
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+        with:
+          ref: continuous-benchmarking
```

Comment:
Here the ref: means the branch that should be checked out. I think it needs to be changed here.

Reply:
okay, I changed it now. But for some reason the check is queded and has not been executed yet. Strange... HOnestly, i have no idea what I am doing 😅